week1/homework/src/responses/sendIndexPage.js

```diff
@@ -0,0 +1,17 @@
+function sendIndexPage(response) {
```

Comment:
I am not sure creating a landing page was part of the requirements for this homework.  Please remove.